app/views/pattern_library/index.slim

```diff
@@ -34,6 +34,7 @@
   == pl_component 'TableSimple', { header: 'Name', value: 'Solange', headerB: 'Other', valueB: 'Jay' }
   == pl_component 'TabsMenu', { items: [ { title: 'Short Form Application', url: '/pattern_library?url1=true' }, { title: 'Supplemental Information', url: '/pattern_library?url2=false' }, { title: 'Other Information', url: '/pattern_library?url3=false' }], currentUrl: request.original_fullpath }
   == pl_component 'MultiSelectWrapper', { items: [{value: 'item1', label: 'Item1'}, {value: 'item2', label: 'Item2'}]}
+  == pl_component 'MultiSelectWrapper', { height: 'small', items: [{value: 'item1', label: 'Item1'}, {value: 'item2', label: 'Item2'}]}
```

Comment:
this kinda ends up just looking like you accidentally put the same thing twice, do we need to have both in the pattern lib? Or could you have the wrapper have 2 of them, one with each height and labels? I don't think we have a great pattern for showing variations on things unfortunately

Reply:
removed the small variation. I think it's okay to just ignore variations for now until we migrate to storyboard or w/e